yt_dlp/extractor/afreecatv.py

```diff
@@ -229,16 +186,20 @@ def _real_extract(self, url):
             raise ExtractorError(
                 'Video %s has been deleted' % video_id, expected=True)
 
-        station_id = self._search_regex(
-            r'nStationNo\s*=\s*(\d+)', webpage, 'station')
-        bbs_id = self._search_regex(
-            r'nBbsNo\s*=\s*(\d+)', webpage, 'bbs')
         video_id = self._search_regex(
-            r'nTitleNo\s*=\s*(\d+)', webpage, 'title', default=video_id)
+            r'vod.afreecatv.com/player/(\d+)/', webpage, 'title', default=video_id)
```

Comment:
Minor regex fix...but is this `video_id = ...` even necessary though? The `video_id` is always matched from the URL, yeah? Are there any tests/examples that need this?
```suggestion
            r'vod\.afreecatv\.com/player/(\d+)/', webpage, 'title', default=video_id)
```